tools/exekall/exekall/engine.py

```diff
@@ -2401,6 +2401,10 @@ class UnboundMethod:
     .. note:: It is generally assumed that if a given method is wrapped in an
         :class:`UnboundMethod`, all subclasses will also have that method
         wrapped the same way.
+
```

Comment:
This commit is now in master, so it should be removed

Reply:
OK.